### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by EdProsser**
This is a work in progress but is mostly here for review and comments. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,165 +1,181 @@
...
+**To connect to the local Fabric instance using the default identity:**
+
+The local instance of Hyperledger Fabric has a default gateway identity: **Admin@org1.example.com**.
+
+To connect using the default gateway identity:
+
+1. After starting the local Fabric instance, expand the **local_fabric** heading in the **Fabric Gateways** pane.
+2. Click the identity listed **Admin@org1.example.com**.
+
+After connecting
```
Comment (caroline-church) [inline README.md]: what comes next???

File: README.md [hunk 2]
```diff
@@ -169,36 +185,52 @@ Debugging your smart contract allows you to run through the smart contract trans
...
+#### Export a smart contract package
 
-> ⚠ Please note: v0.3.0 has restructured fabric connections to fabric gateways, so you will need to add any connections created with previous releases again in order to use them. 
-
-**When using the pre-configured local instance of Hyperledger Fabric named `local_fabric`, the extension will automatically pull and use the correct Docker images.**
+Right-click a package and select the `Export Package` option.  You can select where the package is exported to. Use this to take the packages you have made and deploy them somewhere else, using other tools, such as the Fabric CLI (`peer chaincode install` command).
```
Comment (caroline-church) [inline README.md]: should we also add import packages hear too??

File: README.md [hunk 3]
```diff
@@ -1,165 +1,181 @@
...
 
 You will need the following installed in order to use the extension:
 - [VSCode version 1.31 or greater](https://code.visualstudio.com)
 - [Node v8.x or greater and npm v5.x or greater](https://nodejs.org/en/download/)
 - [Yeoman (yo) v2.x](http://yeoman.io/)
 - [Docker version v17.06.2-ce or greater](https://www.docker.com/get-docker)
 - [Docker Compose v1.14.0 or greater](https://docs.docker.com/compose/install/)
-
-> ⚠ Please note: From version 0.1.0+, your smart contract package.json should depend on at least fabric-contract@1.4.0-beta2. This is only required for smart contracts not created using version 0.1.0+ of this extension.
+- Windows 10 Enterprise, Pro, or Education at 1607 Anniversary Update or later, Linux, or Mac are currently the only supported operating systems.
```
[the remark this answers is not in the document]
Comment (EdProsser) [reply README.md]: If this information isn't correct, it was in the "Supported Operating Systems" section at the bottom, now combined with the pre-reqs. The info it replaces is in another bit of the pre-reqs.

[the remark this answers is not in the document]
Comment (caroline-church) [reply README.md]: it was the "Education at 1607 Anniversary" bit i was whating at

**Comment by heatherlp** [comment]
Converting this to a new one

**Pull request closed without merging**